app/views/inthenewscommentaries/home.html.erb

```diff
@@ -3,47 +3,45 @@
 <h2>Commentaries</h2>
 
 <div class="row">
+  <div class="col-md-3">
+    <h3>Browse</h3>
+    <div class="browse_categories">
+      <%= link_to "Browse All", inthenews_commentaries_browse_path, 
+        class: "display_callout browse_all" %>
+        <%= render partial: "inthenews/browse_home_display", locals: {
+        path: "inthenews_commentaries_browse_facet_path"
+      } %>
+    </div>
+  </div>
 
-  <div class="col-md-5">
-  
+  <div class="col-md-6">
     <%= render partial: "inthenews/search_text_field", locals: {
       url: inthenews_commentaries_search_path,
       label: "Search Commentaries"
     } %>
-    <%= link_to "Browse All Commentaries", inthenews_commentaries_search_path %>
-  </div>
-  <div class="col-md-7">
-
-    <h3>Commentary Authors</h3>
-    <%= render partial: "inthenews/group_count_table", locals: {
-      facet: "creator.name",
-      path: "inthenews_commentaries_search_path"
-    } %>
+    <p><%= link_to "Advanced Search/View All Results", inthenews_commentaries_search_path %></p>
 
-    <h3>Regarding Events</h3>
-    <%= render partial: "inthenews/group_count_table", locals: {
-      facet: "keywords",
-      path: "inthenews_commentaries_search_path"
-    } %>
+   <%# <br/>
```

Comment:
I would prefer just to remove this, if we're not using it.  Easy enough to add back in later!